src/auth/GetWindowsCert.ts

```diff
@@ -0,0 +1,29 @@
+import * as ca from "win-ca";
+import * as caApi from "win-ca/api";
+import * as crypt from "crypto";
+
+const useWindowsCertsThumbprint = (id: string) => {
+  const thumbprint = (cert: string) => {
+    var shasum = crypt.createHash("sha1");
+    shasum.update(Buffer.from(cert, "base64"));
+    return shasum.digest("hex").toUpperCase();
+  };
+
+  const list: string[] = [];
+
+  caApi({
+    store: ["My"],
+    ondata: list,
+  });
+
+  list.forEach((cert) => {
+    const certThumbprint = thumbprint(cert);
+    if (certThumbprint === id) {
+      const toPEM = ca.der2(ca.der2.pem);
+      const pem = toPEM(cert);
+      return pem;
+    }
+  });
+};
+
+export default useWindowsCertsThumbprint;
```

Comment:
Let's reference it in the `login` command where we allow users to specify a certificate. I would suggest that the existing `certificateFile` becomes optional and user can specify either `certificateFile` or a new option named `certificateName` that would allow them to select a certificate from the store (we need to make it clear in the command's help and also mention that it's supported only on Windows).

Reply:
Added seperate command, as both will rely on Thumbprint to pull cert either from file, or from Windows Cert Store.